packages/utils/src/saml/pre-prod-tester.ts

```diff
@@ -397,15 +423,17 @@ async function queryDQ(dqRequest: OutboundDocumentQueryReq): Promise<OutboundDoc
       samlCertsAndKeys,
     });
 
-    const response = await sendSignedDQRequests({
-      signedRequests: xmlResponses,
+    const response = await sendSignedDqRequest({
+      request: xmlResponses[0],
       samlCertsAndKeys,
       patientId: dqRequest.patientId,
       cxId: dqRequest.cxId,
+      index: 0,
     });
 
-    return processDQResponse({
-      dqResponse: response[0],
+    return processDqResponse({
+      response,
+      attempt: 0,
```

Comment:
`attempt` is not being used anymore, is it?

Reply:
I should've commented on the actual function, not where its called - from what I can tell, the only place that calls this fn inside the codebase (not util) doesn't send the attempt, so we might as well remove that param?

Reply:
removed.